test/membrane_rtmp_plugin/rtmp_source_bin_test.exs

```diff
@@ -0,0 +1,84 @@
+defmodule Membrane.RTMP.SourceBin.Test do
+  use ExUnit.Case
+  import Membrane.Testing.Assertions
+
+  require Logger
+
+  alias Membrane.RTMP.Source.TcpServer
+  alias Membrane.Testing
+
+  @input_file "test/fixtures/testsrc.flv"
+  @port 9009
+  @local_ip "127.0.0.1"
+  @rtmp_stream_url "rtmp://#{@local_ip}:#{@port}/"
+
+  test "Check if the stream started and that it ends" do
+    test_process = self()
+
+    options = %TcpServer{
+      port: @port,
+      listen_options: [
+        :binary,
+        packet: :raw,
+        active: false,
+        ip: @local_ip |> String.to_charlist() |> :inet.parse_address() |> elem(1)
+      ],
+      socket_handler: fn socket ->
+        options = [
+          module: Membrane.RTMP.Source.Test.Pipeline,
+          custom_args: [socket: socket, test_process: test_process],
+          test_process: test_process
+        ]
+
+        Testing.Pipeline.start_link(options)
+      end
+    }
+
+    {:ok, _tcp_server} = TcpServer.start_link(options)
+    ffmpeg_task = Task.async(&start_ffmpeg/0)
+
+    pipeline = await_pipeline_started()
+
+    assert_pipeline_playback_changed(pipeline, :prepared, :playing)
```

Comment:
Would be great if you could know the duration of `testsrc.fflv` and make sure that you receive all membrane buffers representing audio and video (assert on them in a loop, make sure the timestamps are ok etc.).